### PR TITLE
CONTRIBUTING: Add link to community.html

### LINKED ISSUE #109: CONTRIBUTING: Add link to community.html
CONTRIBUTING.md needs a link to https://intel.github.io/dffml/community.html

### PR DESCRIPTION
**Pull request by NeerajBhadani**
fixes #109

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,5 +55,5 @@ python3.7 -m http.server --directory htmlcov/ 8080
 
 ## Community
 
-- Join the community. Ask questions, share ideas, request features and what not.
+- Join the [Community][https://intel.github.io/dffml/community.html]. Ask questions, share ideas, request features and what not.
```
Comment (pdxjohnny) [inline CONTRIBUTING.md]: Markdown links need `[]` followed by `()`, this should be

```
[Community](https://intel.github.io/dffml/community.html)
```

Also, can you make these two bullet points

```
- Join the [Community](https://intel.github.io/dffml/community.html). Ask questions, share ideas, request features and what not.
- Join our [gitter](https://gitter.im/dffml/community) room and say Hi!
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   help, they also accept the `CLI_FORMATTER_CLASS` property.
 - CSV source now has `entry_point` decoration
 - JSON source now has `entry_point` decoration
+- Added Community link in CONTRIBUTING.md.
```
[the remark this answers is not in the document]
Comment (NeerajBhadani) [reply CHANGELOG.md]: @pdxjohnny  fixed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (NeerajBhadani) [reply CONTRIBUTING.md]: @pdxjohnny  updated.

Comment (pdxjohnny) [reply CONTRIBUTING.md]: Thank you!

**Pull request merged**